about.css

```diff
@@ -0,0 +1,582 @@
+@import url("https://fonts.googleapis.com/css2?family=COCOGOOSE:wght@400;500;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+* {
+  list-style: none;
+  text-decoration: none;
+  margin: 0%;
+  box-sizing: border-box;
+  font-family: 'cocogoose', sans-serif;
+}
+
+header {
+  box-shadow: 0 3px 10px 0 #4b4b4b73;
+  width: 100%;
+}
+
+.top-nav {
+  display: none;
+}
+
+.bottom-nav {
+  display: none;
+  background-color: #fff;
+  padding-bottom: 10px;
+  padding-top: 10px;
+}
+
+.hamburger {
+  padding-left: 20px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  background-color: #fff;
+  font-size: 40px;
+}
+
+.mobile-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 20px;
+  padding-top: 15px;
+  font-size: 35px;
+  padding-bottom: 15px;
+  background-color: #d3d3d3;
+  opacity: 0.7;
+}
+
+#mobile-menu {
+  visibility: hidden;
+  background-color: #fff;
+  height: 100vh;
+  top: 0;
+  width: 100%;
+  position: fixed;
+}
```

Comment:
Great job @NgozikaNwocha 🤝

- [ ] [Optional] Kindly remove that section I circled in the image below. It is not supposed to appear on your popup.
![popup](https://user-images.githubusercontent.com/63917725/181621711-27015e8e-cc01-4de3-b1cb-7d9bdddfae57.PNG)